.travis.yml

```diff
@@ -1,16 +1,16 @@
 language: java
 sudo: false # faster builds
 
-install: mvn clean install -DskipTests
-
-script: mvn clean verify
+addons:
+  sonarcloud:
```

Comment:
I must be blind 🙈, but where can I see the results of this?

Reply:
https://sonarcloud.io/project/issues?id=parsingdata_metal&pullRequest=270&resolved=false